test/channels/retro_channel_test.exs

```diff
@@ -207,4 +220,20 @@ defmodule RemoteRetro.RetroChannelTest do
       assert participation.vote_count == 1
     end
   end
+
+  describe "pushing a `submit_vote` event to the socket when the participation has 5 votes" do
+    setup [:persist_user_for_retro, :persist_idea_for_retro, :persist_maxed_out_participation_for_retro, :join_the_retro_channel]
+
+    @tag user: @mock_user
+    @tag idea: %Idea{category: "sad", body: "JavaScript"}
+    test "results in the broadcast of the voted on idea and updated participation to all connected clients", %{socket: socket, idea: idea, user: user, participation: participation} do
```

Comment:
the test language is a little off here. this seems to be testing that the vote count isn't incremented beyond 5. let's update it